# Orbit Browser patch: chrome/browser/content/browser/hiddenWindowMac.xhtml
# platforms: mac
#
# menu_newAIWindow, hidden in the Mac's windowless menu bar.
# menu_newClassicWindow, hidden in the Mac's windowless menu bar.
# feedbackPage, hidden in the Mac's windowless menu bar.
# helpSwitchDevice, hidden in the Mac's windowless menu bar.
# help_reportBrokenSite, hidden in the Mac's windowless menu bar.

--- a/chrome/browser/content/browser/hiddenWindowMac.xhtml
+++ b/chrome/browser/content/browser/hiddenWindowMac.xhtml
@@ -372,10 +372,10 @@
       <menuitem id="menu_newNavigator"
                 key="key_newNavigator"
                 command="cmd_newNavigator" data-l10n-id="menu-file-new-window"/>
-      <menuitem id="menu_newAIWindow"
+      <menuitem id="menu_newAIWindow" hidden="true"
                 command="Tools:AIWindow"
                 data-l10n-id="menu-file-new-ai-window"/>
-      <menuitem id="menu_newClassicWindow"
+      <menuitem id="menu_newClassicWindow" hidden="true"
                 command="Tools:ClassicWindow"
                 data-l10n-id="menu-file-new-classic-window"/>
       <menuitem id="menu_newPrivateWindow"
@@ -718,12 +718,12 @@
                 data-l10n-id="menu-get-help"
                 appmenu-data-l10n-id="appmenu-get-help"
                 />
-      <menuitem id="help_reportBrokenSite"
+      <menuitem id="help_reportBrokenSite" hidden="true"
                 command="cmd_reportBrokenSite"
                 data-l10n-id="menu-report-broken-site"
                 disabled="true"
                 appmenu-data-l10n-id="appmenuitem-report-broken-site"/>
-      <menuitem id="feedbackPage"
+      <menuitem id="feedbackPage" hidden="true"
                 data-l10n-id="menu-help-share-ideas"
                 appmenu-data-l10n-id="appmenu-help-share-ideas"/>
       <menuitem id="helpSafeMode"
@@ -742,7 +742,7 @@
                 data-l10n-id="menu-help-not-deceptive"
                 appmenu-data-l10n-id="appmenu-help-not-deceptive"
                 hidden="true"/>
-      <menuitem id="helpSwitchDevice"
+      <menuitem id="helpSwitchDevice" hidden="true"
                 data-l10n-id="menu-help-switch-device"
                 appmenu-data-l10n-id="appmenu-help-switch-device"/>
       <menuseparator id="aboutSeparator"/>
